src/SEO.astro

```diff
@@ -47,7 +47,7 @@ if (openGraph) {
 ---
 { title ? <title>{title}</title> : null }
 
-{ canoncial ? (
+{ canonical ? (
```

Comment:
Thanks for catching this! The variable name needs to be changed in props destructuring as well though 😃